client/util/util_test.go

```diff
@@ -27,3 +28,12 @@ func TestLoadSignerBinaryPath(t *testing.T) {
 		t.Errorf("Expected path is %q, got: %q", want, path)
 	}
 }
+
+func TestGetConfigFilePathFromEnv(t *testing.T) {
```

Comment:
Can you add a test case that checks that an explicit filepath sent to the Cred function is prioritized over the env variable?

Reply:
client_test.go would probably be where you'd add the test to test the behavior Carl mentioned.

Reply:
Okay, I will work on that. Thank you!